### DISCUSSION TITLE
Password Confirmation

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5503] [Q&A] [answered]
I am create client blueprint and add some fields like : name, email, phone and password. I want to add password confirm field to check the password again to complete the process. How to make this please ?

**Comment by jasonvarga** [comment, marked as answer]
In your other discussion (https://github.com/statamic/cms/discussions/5505) we explain that you should be using actual users. 

However, this is an example of where the [`confirmed` validation rule](https://laravel.com/docs/9.x/validation#rule-confirmed) would be useful.

**Reply by HazemMohamed252** [reply]
Thanks for help
